contextualhelp/data.json

```diff
@@ -1661,59 +1734,65 @@
+            "body": "It is a 6-digit code that you set the first time you log in to IO and that allows you to enter the app without having to enter SPID or CIE credentials each time. It should also be entered to view sensitive information and to confirm certain operations."
           },
           {
             "title": "I forgot the unlock code. What can I do?",
-            "body": "Press 'Did you forget the unlock code?', insert your SPID or CIE credentials, and enter the new code."
+            "body": "When you enter the app, on the code entry screen press ‘Forgot your unlock code?’, authenticate with SPID or CIE and enter the new code."
           },
           {
             "title": "How can I change the unlock code?",
-            "body": "Select 'Change the unlock code', insert your SPID or CIE credentials, and enter the new code."
+            "body": "If you remember the unlock code but want to change it, select ’Change unlock code,' authenticate with SPID or CIE and enter the new code."
           },
           {
             "title": "What is biometric recognition for?",
-            "body": "You can use biometric recognition, when enabled, in place of the unlock code. Just like the unlock code, this authentication mode remains active for 30 days after your last login with SPID or CIE. After that time frame, you will need to re-authenticate."
+            "body": "If enabled, you can use biometric recognition instead of the unlock code."
           },
           {
             "title": "How can I enable biometric recognition?",
-            "body": "If your phone supports biometric recognition, enable it from the system preferences. Then, go to the 'Profile' section of the IO app and enable it there too."
-          },
-          {
-            "title": "How can I disable biometric recognition?",
-            "body": "Go to the 'Profile' section of the IO app. Biometric recognition will already be turned off if your device does not support it, or if it is disabled in the system preferences."
+            "body": "If your phone supports biometric recognition, enable it from system preferences. Once done, also enable it from the 'Settings' of the IO app."
           }
         ]
       },
       {
         "route_name": "PROFILE_PRIVACY_MAIN",
         "title": "Your privacy settings",
-        "content": "Here you can find information about the processing of your personal data and IO's terms of use. More specifically, you can read the Privacy Policy, download a copy of the data associated with your account or delete it and give or withdraw permission to the app to collect your usage data.",
+        "content": "Here you can find all the information regarding the processing of your personal data. You can also request a copy of the data associated with your profile or request that it be deleted.",
```

Comment:
The comma (`,`) at the end is wrong. Maybe it's shifted from somewhere else during editing?

Reply:
Typing error, it was a dot